py-jupyter.md

```diff
@@ -0,0 +1,249 @@
+## The Jupyter notebook {#py-the-jupyter-notebook}
+
+Although the Python interpreter is very powerful,
+it is commonly bundled with other useful tools and interfaces
+specifically designed for exploratory data analysis.
+One such interface is the JupyterLab from the Jupyter project.
+
+Jupyter originates from a project called IPython,
+an effort to make Python development more interactive.
+Since its inception,
+the scope of the project expanded to include additional programming languages,
+such as Julia, Python, and R,
+so the name was changed to "Jupyter" as a reference to these core languages.
+Today,
+Jupyter supports many more languages,
+but we will be using it only for Python code.
+Specifically,
+we will be using the JupyterLab IDE and work in Jupyter notebooks,
+which allows us to easily take notes about our analysis
+and view plots within the same document where we code.
+This facilitates sharability and reproducibility of analyses,
+and the notebook interface is easily accessible through any web browser
+as well as exportable as a PDF or HTML page.
+
+Open JupyterLab by running `juptyerlab` from the terminal,
+by finding it in the `Anaconda navigator`,
+or directly from the start menu if your operating system supports it.
+This should output some text in the terminal and open new tab in your default browser.
+The JupyterLab launcher page is shown on startup,
+to create a new notebook,
+click the big Python tile under the notebook section or go to `File --> New --> Notebook`.
+If you click on the File Browser in the sidebar to the left (the topmost folder icon),
+you will see that a new file has been created in the current directory.
+The File Browser can be used to open existing notebooks and other file types.
+The new notebook is named `Untitled.ipynb` by default.
+Right-click the name in the notebook tab,
+and change it to `getting-started.ipynb`.
+
+The notebook is divided into cells.
+Initially, there will be a single input cell.
+ <!--com: TODO add screenshot? -->
+You can type Python code directly into this cell,
+just as we did before in the IPython terminal.
+To run the code,
+click the play button in the toolbar or press <kbd>Shift</kbd> + <kbd>Enter</kbd>.
+
+```{python}
+4 + 5
+```
+
+The output will appears just under the input cell.
+By default,
+the next existing cell is selected,
+or a new empty one is created if there is no next cell.
+You can press <kbd>Ctrl</kbd> + <kbd>Enter</kbd> to stay on the current cell after running code.
+
+In an input cell,
+you can split the code across several lines as needed.
+
+```{python}
+a = 4
+a * 2
+```
+
+You might have noticed that there is a little counter on the left of each cell.
+This counter keeps track of in which order the cells were executed,
+so that you are aware if they cells have been run out of sequential order.
+The counter symbol changes to an `*` when a computation is running,
```

Comment:
Isn't this correct though, since it is the pronunciation that decides? Likewise, `an hour` and `a university` would appear visually strange at first, but we're so used to it that we don't react.